app/src/main/java/fr/neamar/kiss/result/AppResult.java

```diff
@@ -46,15 +46,20 @@ public AppResult(AppPojo appPojo) {
 
     @Override
     public View display(final Context context, int position, View v) {
-        if (v == null)
-            v = inflateFromId(context, R.layout.item_app);
+        if (v == null) {
```

Comment:
this won't work

List View recycle Results for performance, so you're inflating something that might be reused for another app with tags (or without tags). This might even crash when the system will reuse a tag-less holder for a tagged app and get a NullPointerException

You need to either define a completely new Result type, or better to use a linearlayout and Visibility.GONE with the correct gravity to get the result you're looking for :)